pkg/sql/tests/rsg_test.go

```diff
@@ -83,18 +83,52 @@ type verifyFormatDB struct {
 		syncutil.Mutex
 		// active holds the currently executing statements.
 		active map[string]int
+		// lastStmtBuffer contains the last set of statements executed
+		// within a ring buffer.
+		lastStmtBuffer [200]string
```

Comment:
is 200 too much? ultimately up to what you think would help debug these types of tests